src/swap/components/SwapMessage.test.tsx

```diff
@@ -93,4 +93,104 @@ describe('SwapMessage', () => {
     const messageDiv = screen.getByTestId('ockSwapMessage_Message');
     expect(messageDiv).toHaveClass(customClass);
   });
+
+  test('sets isMissingRequiredFields to true when from.amount is missing', () => {
```

Comment:
We usually use `it('should `, althought `test` is the same.

I just can tell those are CoPilot generated, and than I question if we triple check them :)

Reply:
yeah for some reason the other tests here were using `test` but i'll switch them all to `it`.